Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -1593,6 +1593,14 @@ WebIDL::ExceptionOr<void> HTMLInputElement::set_type(String const& type)
     return set_attribute(HTML::AttributeNames::type, type);
 }
 
+bool HTMLInputElement::should_have_cursor() const
+{
+    if (type_state() == TypeAttributeState::SubmitButton || type_state() == TypeAttributeState::ResetButton || type_state() == TypeAttributeState::Button)
```

Comment:
We have a `first_is_one_of()` function that can make things like this easier to read. eg:
```c++
if (first_is_one_of(type_state(), TypeAttributeState::SubmitButton, TypeAttributeState::ResetButton, TypeAttributeState::Button))
```